Manifests/ManifestsApple/com.apple.MCX-MobileAccounts.plist

```diff
@@ -153,12 +157,72 @@ The payload organization for a payload need not match the payload organization i
 			<key>pfm_name</key>
 			<string>cachedaccounts.askForSecureTokenAuthBypass</string>
 			<key>pfm_title</key>
-			<string>Ask For Secure Token Auth Bypass</string>
+			<string>Ask for Secure Token Auth Bypass</string>
 			<key>pfm_type</key>
 			<string>boolean</string>
 			<key>pfm_macos_min</key>
 			<string>10.13.5</string>
 		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<integer>-1</integer>
+			<key>pfm_description</key>
+			<string>Mobile account lifetime before automatic removal in an integer value representing seconds, where -1 means never and 0 means at next login</string>
+			<key>pfm_description_reference</key>
+			<string>The minimum number of seconds a mobile account can exist before an automatic attempt is made to remove the mobile account.</string>
+			<key>pfm_name</key>
+			<string>cachedaccounts.expiry.delete.disusedSeconds</string>
+			<key>pfm_title</key>
+			<string>Cached Accounts Expiry</string>
+			<key>pfm_type</key>
+			<string>integer</string>
+			<key>pfm_range_min</key>
+			<integer>-1</integer>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<false/>
+			<key>pfm_description</key>
+			<string></string>
+			<key>pfm_description_reference</key>
+			<string>If true, allows the user to stop the prompts about mobile account creation every time the user logs in. This key is only valid if com.apple.cachedaccounts.WarnOnCreate is set to true.</string>
+			<key>pfm_name</key>
+			<string>cachedaccounts.WarnOnCreate.allowNever</string>
+			<key>pfm_title</key>
+			<string>Allow Users to Stop Mobile Account Creation Prompts</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
```

Comment:
Suggesting the addition of the following code.  Reason being is the only time this key should be used is in conjunction with `com.apple.cachedaccounts.WarnOnCreate` when it's set to `true`.  If someone doesn't want users to be prompted about Mobile Account creation, then `com.apple.cachedaccounts.WarnOnCreate` set to false and therefore exclude`com.apple.cached.accounts.allowNever` if added.

```
<key>pfm_exclude</key>
<array>
	<dict>
		<key>pfm_target_conditions</key>
		<array>
			<dict>
				<key>pfm_present</key>
				<false/>
				<key>pfm_range_list</key>
				<array>
					<false/>
				</array>
				<key>pfm_target</key>
				<string>com.apple.cachedaccounts.WarnOnCreate</string>
			</dict>
		</array>
	</dict>
</array>
```

Reply:
FYI, that code ^ has been tested successfully.

Reply:
Seems like a good idea. Added via commit `dca5fff`.